El Carto.js

```diff
@@ -0,0 +1,182 @@
+var count = 1;
```

Comment:
You've got everything except calculating total price and putting it on the page.  The create new item features works.